### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by DThompson4**
Implemented a Siamese Neural Network to classify the ADNI dataset.  The SNN trained very well, achieving high validation accuracy. The classifier also resulted in well trained model with high validation accuracy. However had issues when predicting, poor accuracy when predicting compared to the validation accuracy encountered in training.

### CODE REVIEW

**Comment by nathasha-naranpanawa** [comment]
- Repo containes Readme files from two other students (?)
- Commit messages can be more descriptive
- Readme is OK
- Not sure if loss plot is for training or validation. Only one trend shown
- No predcition examples shown
- Performance is good. Loss function for Siamese implemented correctly.

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage 
Module present
Commenting
No Data leakage
Difficulty: Hard
## Commit Log
Meaningful commit messages -1
Progressive commits used
## Documentation
ReadMe acceptable but output missing and plot unlabelled -2
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Feedback required, remove changes to other students Readme files -2
Request Description short -1

**Comment by DThompson4** [comment]
Removed changed files of other students, I put a post ed to explain what happened there. Inadvertently changed the gitignore so also rolled that back as well.
